packages/manager/modules/pci/src/projects/project/project.routing.js

```diff
@@ -169,6 +175,14 @@ export default /* @ngInject */ ($stateProvider) => {
           projectId,
         });
       },
+
+      vouchersCreditInfo: /* @ngInject */ (PciProject, projectId) => () => {
+        return PciProject.getVouchersCreditInfos(projectId)
+          .then((data) => {
+            return data.data || [];
+          })
```

Comment:
```suggestion
          .then(({ data = []}) => {
            return data;
          })
```

Reply:
I removed the ' || []' since I had added the .catch (...) clause.